tests/screens/ProfileScreen.test.tsx

```
@@ -31,117 +93,243 @@ jest.mock('@tanstack/react-query', () => ({
+    mutate: jest.fn(), // Mock function for triggering mutations
+    mutateAsync: jest.fn(), // Mock function for async mutations
+    isLoading: false, // Default loading state
+    isError: false, // Default error state
+    isSuccess: false, // Default success state
+    error: null, // Default error value
+  }),
+}));
+
+export const TestWrapper = ({ children }: { children: ReactNode }) => {
+  return <AuthProvider>{children}</AuthProvider>;
+};
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)